include/sys/zfs_znode.h

```diff
@@ -242,9 +242,9 @@ zfs_enter_verify_zp(zfsvfs_t *zfsvfs, znode_t *zp, const char *tag)
 
 typedef struct znode_hold {
 	uint64_t	zh_obj;		/* object id */
-	kmutex_t	zh_lock;	/* lock serializing object access */
 	avl_node_t	zh_node;	/* avl tree linkage */
-	zfs_refcount_t	zh_refcount;	/* active consumer reference count */
+	kmutex_t	zh_lock;	/* lock serializing object access */
```

Comment:
Why were the structure members reordered? Was this due to the structure packing becoming holey after the type change?

Reply:
To put zh_obj and zh_node into the same cache line for easier AVL tree traverse.